Lib/test/test_capi/test_getargs.py

```diff
@@ -151,6 +151,8 @@ class TupleSubclass(tuple):
 class DictSubclass(dict):
     pass
 
+m = memoryview(bytearray(b'noncontig'))
+NONCONTIG_WRITABLE = m[::-2]
```

Comment:
Use also this for tests:
```python
NONCONTIG_READONLY = memoryview(b'noncontig')[::-2]
```
Some tests may produce different errors for writeable and read-only buffers.